liquidator/liquidator.js

```diff
@@ -125,20 +135,14 @@ class Liquidator {
 
     this.logger.debug({
       at: "Liquidator",
-      message: "Scaling down collateral threshold for liquidations",
+      message: "Checking for under collateralized positions",
       inputPrice: price.toString(),
       scaledPrice: scaledPrice.toString(),
       empCRRatio: this.empCRRatio.toString(),
       maxCollateralPerToken: maxCollateralPerToken.toString(),
       crThreshold: this.crThreshold
     });
 
-    this.logger.debug({
```

Comment:
having these two logs back to back was redundant. They literally followed one another and had some overlapping information. I chose to join them for brevity.

Reply:
agreed